sw/device/silicon_creator/lib/shutdown_unittest.cc

```diff
@@ -245,9 +245,9 @@ static_assert(ARRAYSIZE(kDefaultAlertClassification) <=
               "The default alert classification must be less than or equal to "
               "the number of reserved OTP words");
 
-static_assert(
-    kTopEarlgreyAlertIdLast < ARRAYSIZE(kDefaultAlertClassification),
-    "The number of alert sources must be smaller than the alert classification");
+static_assert(kTopEarlgreyAlertIdLast < ARRAYSIZE(kDefaultAlertClassification),
```

Comment:
Hmm I think clang-format has gotten over excited and made a bunch of unrelated changes to shutdown. Clang format used to be disabled on this file (by accident - the `// clang-format on` directive was appended to the end of a macro and ignored). Let me run clang-format over the shutdown code to make this PR cleaner.

Reply:
sounds good. I merged your PR and will rebase on top.